docs/source/scale/sagemaker.rst

```diff
@@ -276,9 +274,7 @@ jobs that rely on the ``Estimator`` base class: For a full list of
 The GConstruct job will launch a SageMaker Processing job that relies on
 the ``Processor`` base class, so its arguments are different,
 e.g. ``volume_size_in_gb`` for the ``Processor`` vs. ``volume_size`` for
-the ``Estimator``.
-
-For a full list of ``Processor`` parameters see the `SageMaker Processor documentation.
+the ``Estimator``. For a full list of ``Processor`` parameters see the `SageMaker Processor documentation.
```

Comment:
A minor language suggestion "parameters see the" -> "parameters, see the"

Reply:
changed.